### PR TITLE
[IMP] project: move the attachments section below the description

### PR DESCRIPTION
**Pull request by mjvi-odoo**
Description of the issue/feature this PR addresses:

Desired behavior after PR is merged:

- In the portal task template the 'attachments' section below the description.

task-3969566

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/views/project_portal_project_task_templates.xml [hunk 1]
```diff
@@ -255,12 +255,12 @@
                             </div>
 
                             <div class="row" t-if="task.description or task.attachment_ids">
-                                <div t-if="not is_html_empty(task.description)" t-attf-class="col-12 mb-4 mb-md-0 {{'col-lg-6' if task.attachment_ids else 'col-lg-12'}}">
+                                <div t-if="not is_html_empty(task.description)" t-attf-class="mb-4 mb-md-0">
```
Comment (ppr-odoo) [inline addons/project/views/project_portal_project_task_templates.xml:258]:
```suggestion
                                <div t-if="not is_html_empty(task.description)" class="mb-4 mb-md-0">
```

File: addons/project/views/project_portal_project_task_templates.xml [hunk 2]
```diff
@@ -255,12 +255,12 @@
...
                                     <h5 class="mb-1">Description</h5>
                                     <hr class="mt-1 mb-2"/>
                                     <div class="py-1 px-2 bg-100 small overflow-auto table-responsive" t-field="task.description"/>
                                 </div>
-                                <div t-if="task.attachment_ids" t-attf-class="col-12 o_project_portal_attachments {{'col-lg-6' if task.description else 'col-lg-12'}}">
+                                <div t-if="task.attachment_ids" t-attf-class="o_project_portal_attachments">
                                     <h5 class="mb-1">Attachments</h5>
```
Comment (ppr-odoo) [inline addons/project/views/project_portal_project_task_templates.xml:264]: same here

**Review by xavierbol — COMMENTED** [review]
LGTM, thanks for your work. 🙂 

robodoo R+

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**